Gemfile

```diff
@@ -3,9 +3,9 @@ source 'https://rubygems.org'
 gem 'bundler', '>= 1.8.4'
 gem 'rails', '~> 5.1.0'
 
-gem 'acts-as-taggable-on', github: 'mbleigh/acts-as-taggable-on'
+gem 'acts-as-taggable-on', git: 'https://github.com/mbleigh/acts-as-taggable-on.git'
```

Comment:
Why die you change those source specifications?

Reply:
Because the stable version of bundler complains about those source specifications as beeing unsecure. It tries to clone them by using a http connection. Looks like those source specification will be deprecated in a [future release](https://github.com/bundler/bundler/pull/5705)

Reply:
I would prefer a solution [like in Rails](https://github.com/rails/rails/blob/master/Gemfile#L5).